py/selenium/webdriver/support/expected_conditions.py

```diff
@@ -95,6 +95,8 @@ def __call__(self, ignored):
 
 
 def _element_if_visible(element, visibility=True):
+    if isinstance(element, str) or isinstance(element, dict):
+           raise StaleElementReferenceException("Invalid locator")
```

Comment:
This problem only happen in invisibility_of_element_located, no have problem that return true or false, but  the problem is that it throws an exception in the code and stop my test.